analyses/focal-cn-file-preparation/run-prepare-cn-OpenTarget.sh

```diff
@@ -52,29 +52,35 @@ if [ "$RUN_ORIGINAL" -gt "0" ]; then
     --gtf_file $gtf_file \
     --metadata $histologies_file \
     --filename_lead "cnvkit_annotated_cn" \
-    --seg
+    --seg \
+    --runWXSonly
 
   # Run annotation step for ControlFreeC
   Rscript --vanilla 04-prepare-cn-file.R \
     --cnv_file ${data_dir}/cnv-controlfreec.tsv.gz \
     --gtf_file $gtf_file \
     --metadata $histologies_file \
     --filename_lead "controlfreec_annotated_cn" \
-    --controlfreec
+    --controlfreec \
+    --runWXSonly
 
   filenameLead=("cnvkit_annotated_cn" "controlfreec_annotated_cn")
+  chromosomeType=("autosomes" "x_and_y")
+  runWXSOnly=("wxs" "")
```

Comment:
Is another `runWXSonly` variable needed here ? Can we add as a input logical param --runWXSonly as a  in `rna-expression-validation.R` like we do in `04` to be consistent?

Reply:
Since the file name is hard-coded so in order to do that, I had to change `filenameLead=("cnvkit_annotated_cn" "controlfreec_annotated_cn" "cnvkit_annotated_cn_wxs" "controlfreec_annotated_cn_wxs")
  ` and then added `if(runWXSonly){
  if(!grepl("_wxs_", annotated_cnv_file)){
    Stop("Running for wxs only - no need to run for complete file. Skipping to next.")
  }
}` in `rna_expression_validation.R`. Please check whether that make sense.